localstack/services/ssm/resource_providers/aws_ssm_parameter.py

```diff
@@ -151,12 +151,11 @@ def update(
         ]
         update_config_props = util.select_attributes(model, parameters_to_select)
 
-        # tag handling
         # tag handling
         new_tags = update_config_props.pop("Tags", {})
         self.update_tags(ssm, model, new_tags)
 
-        ssm.put_parameter(Overwrite=True, **update_config_props)
+        ssm.put_parameter(Overwrite=True, Tags=[], **update_config_props)
```

Comment:
Workaround for a regression caused by an upstream change https://github.com/getmoto/moto/pull/7066

Reply:
Yes, `new_tags` are assigned to the SSM parameter is per the conditionals in `self.update_tags()`

This is tested by `test_update_ssm_parameter_tag` (which also caught this issue)